README.md

```diff
@@ -257,25 +213,18 @@ A spicy roll and a great option for vegetarians.
 
 This is a very famous Singaporean dish.
 
-Tian Tian in the Maxwell Food Centre is very famous (and I hear half decent).
-If you like to stand in a long queue, please go there at lunch. If you don't go
-there at lunch, they're unlikely to be open.
+Tian Tian in the Maxwell Food Centre is very famous, although the queues can get long, so arrive early if you go there for lunch.
```

Comment:
I think we should recommend boon tong kee :) I think the chicken rice is better but less affordable.